README.md

```diff
@@ -54,8 +54,8 @@ When developing your extensions, you need to manually enable your extensions wit
 notebook / lab frontend. For lab, this is done by the command:
 
 ```
-jupyter labextension install @jupyter-widgets/jupyterlab-manager --no-build
-jupyter labextension install .
+jupyter labextension develop --overwrite .
+npm run build
```

Comment:
Something that was discussed in https://github.com/jupyter-widgets/widget-cookiecutter/pull/72.

Maybe we'll want to recommend using `yarn`, so it stays consistent with the `jlpm` and doesn't mix both `yarn.lock` and `package-lock.json` files?

Reply:
Sure - should we also change all the scripts in the package.json then?

Reply:
Probably yes. I think it's used in the widget cookiecutter here: 

https://github.com/jupyter-widgets/widget-cookiecutter/blob/969471848da747d5aa562b8ab00988e3117e4ccf/%7B%7Bcookiecutter.github_project_name%7D%7D/js/package.json#L24